src/main/groovy/de/dkfz/eilslabs/batcheuphoria/IntegrationTestStarter.groovy

```diff
@@ -162,35 +183,86 @@ class IntegrationTestStarter {
             log.severe("Not all jobs ${jobList.collect { it.jobID }.join(" ")} were in the proper state: [${listOfStatesToCheck.join(" ")}], got last states [${lastStates.join(" ")}]. Make sure, that your job system is working properly.")
     }
 
-    private static void checkStartup(String[] args) {
-        if (!args || args.size() < 2) {
-            println([
-                    "Call the integration test starter with a range of optione like",
-                    "java ... ${IntegrationTestStarter.class.name} PBS SGE SLURM LSF",
-                    "where the cluster system names are optional but at least one has to be set.",
-                    "Available options are (lower case is accepted):",
-                    " direct",
-                    " pbs",
-                    " sge",
-                    " slurm",
-                    " lsf",
-            ].join("\n"))
+    private static void checkStartup(String[] args, IntegrationTestInput testInput) {
+        def cli = new CliBuilder(usage: '-s "server" -a "account" -rs "rest_server" -ra "rest_account" -c [lsf,pbs] ')
+        cli.h(longOpt: 'help'  , 'usage information'   , required: false           )
+        cli.c(longOpt: 'cluster', 'cluster system (lsf, pbs)', required: true , args: 1 )
+        cli.s(longOpt: 'server', 'server to connect to', required: false  , args: 1 )
+        cli.a(longOpt: 'account', 'user account', required: false  , args: 1 )
+        cli.ra(longOpt: 'restaccount', 'rest account (for lsf required)', required: false  , args: 1 )
+        cli.rs(longOpt: 'restserver', 'rest server (for lsf required)', required: false  , args: 1 )
```

Comment:
this is all so non-speaking... Could you make long names out of it? There are also no comments. Please do one of both.